### PR TITLE
peer tool formulae

### PR DESCRIPTION
**Pull request by Horta**
webpage: https://github.com/PMBio/peer/wiki/

### CODE REVIEW

**Comment by Horta** [comment]
Peer tool install both executable but also a python package but looks like brew does not allow that formulae to copy a file to site-packages folder.

Does anyone know how to allow this to happen?

> -- Installing: /usr/local/Cellar/python/2.7.12/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/_peer.so
> CMake Error at python/cmake_install.cmake:39 (file):
>   file INSTALL cannot copy file
>   "/tmp/peer-20160731-25452-1bklzla/peer/python/_peer.so" to
>   "/usr/local/Cellar/python/2.7.12/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/_peer.so".
> Call Stack (most recent call first):
>   cmake_install.cmake:36 (include)

File: peer.rb [hunk 1]
```diff
@@ -0,0 +1,141 @@
+class Peer < Formula
+  desc "Infer hidden determinants and their effects from gene expression profiles."
+  homepage "https://github.com/PMBio/peer/wiki"
+  url "https://github.com/downloads/PMBio/peer/peer_source_1.3.tar.gz"
+  sha256 "c6051bca8d99097ba774ebb734155d1ff9b5cb6ebcea164de8e8e74c97514f56"
+
+  depends_on "cmake" => :build
+  depends_on "swig" => :build
+  depends_on "eigen" => :build
+  depends_on "python"
```
Comment (dpo) [inline peer.rb]: Should be `:python`

File: peer.rb [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+  depends_on "homebrew/python/numpy"
```
Comment (dpo) [inline peer.rb]: `depends_on "numpy" => :python`

File: peer.rb [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+  
+  patch :DATA
+
+  def install
+    rm_rf "External/Eigen"
+    args = ["-DBUILD_PEERTOOL=1",
+            "-DCMAKE_INCLUDE_PATH=#{HOMEBREW_PREFIX}/include/eigen3",
```
Comment (dpo) [inline peer.rb]: Please use `Formula["eigen"].opt_include`

File: peer.rb [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+            "-DCMAKE_INSTALL_PREFIX:PATH=#{prefix}"]
+    system "cmake", *args
+    system "make", "install", "CPLUS_INCLUDE_PATH=#{HOMEBREW_PREFIX}/include/eigen3"
```
Comment (dpo) [inline peer.rb]: Is this going to install things to the `eigen` include dir?

File: peer.rb [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+  url "https://github.com/downloads/PMBio/peer/peer_source_1.3.tar.gz"
+  sha256 "c6051bca8d99097ba774ebb734155d1ff9b5cb6ebcea164de8e8e74c97514f56"
+
+  depends_on "cmake" => :build
+  depends_on "swig" => :build
+  depends_on "eigen" => :build
+  depends_on "python"
+  depends_on "homebrew/python/numpy"
+  
+  patch :DATA
```
Comment (dpo) [inline peer.rb]: Were those changes submitted upstream?

**Comment by dpo** [comment]
Bump

**Pull request closed without merging**